floris/tools/optimization/scipy/yaw_wind_rose.py

```diff
@@ -171,8 +171,57 @@ def __init__(
             unc_options=unc_options,
         )
 
+        self._get_initial_farm_power()
```

Comment:
I think this makes more sense in reinitialize_opt_wind_rose so it gets called every time the object is updated.